packages/core/src/manifest/params/apply-params.ts

```diff
@@ -3,7 +3,7 @@ import { getModuleOrThrow } from "../../utils/get-module.js";
 
 export function applyParams(params: string[], modules: Module[]) {
   for (const param of params) {
-    const [module, value] = param.split("=", 2);
+    const [module, value] = param.split(/=(.*)/s);
     if (module === undefined || value === undefined) {
```

Comment:
Let's fix this without a regex and also allow empty parameters then...

What about sth. like this

```suggestion
    const index = params.indexOf("=");
    if (index === -1) {
      throw new Error(`Invalid param ${param}. Must be in the form of "module=value" or "imported:module=value"`);
    }

    const module = param.slice(0, index);
    const value = param.slice(index + 1);
```